metrics/src/main/java/tech/pegasys/pantheon/metrics/prometheus/PrometheusMetricsSystem.java

```diff
@@ -60,7 +60,7 @@
 
   private final EnumSet<MetricCategory> enabledCategories = EnumSet.allOf(MetricCategory.class);
 
-  PrometheusMetricsSystem() {}
+  public PrometheusMetricsSystem() {}
```

Comment:
This is deliberately not public.  use `init()`
```suggestion
  PrometheusMetricsSystem() {}
```